osu.Game.Tournament/Components/TournamentIntro.cs

```diff
@@ -287,7 +287,7 @@ private void updateAnimationState()
 
         private void beginAnimation()
         {
-            using (BeginDelayedSequence(200))
+            using (BeginDelayedSequence(1500))
```

Comment:
I have adjusted some parameters of the animation to make them looks less visual distracting, also made them smoother.

This delay is added on purpose, as we have to give our viewers a chance to let them know the position of the block being picked.

Reply:
That's quite duide bro, going to liu binglang